templates/zerver/help/getting-started-with-zulip.md

```diff
@@ -0,0 +1,58 @@
+# Getting started with Zulip
+
+Welcome to Zulip!  Like any new tool, mastering Zulip will take some time,
+but once you get the hang of it, you'll never want to use a chatroom tool
+again.
+
+Zulip is designed around a few key ideas:
+
+- You spend far more time reading communications than writing them.  So
+  Zulip is primarily designed to optimize consuming messages; either reading
+  them, or making an informed decision not to read them.
+- Messages in Zulip are organized into streams and topics.  Streams are
+  similar to chatrooms, IRC channels, or email lists, and determine who
```

Comment:
nit: with the ", and" after email lists, the second clause would have to be an independent clause.  But it's determining who receives a message that is why they are similar, so maybe reword to "Streams are similar to chatrooms, IRC channels, or email lists in that they determine who receives the message"?